installer/src/main/java/com/amazon/aws/partners/saasfactory/saasboost/SaaSBoostInstall.java

```diff
@@ -1313,6 +1313,9 @@ protected String getExistingSaaSBoostStackName() {
         try {
             GetParameterResponse response = ssm.getParameter(request -> request.name("/saas-boost/" + this.envName + "/SAAS_BOOST_STACK"));
             stackName = response.parameter().value();
+        } catch (ParameterNotFoundException paramStoreError) {
+            LOGGER.warn("Parameter /saas-boost/" + this.envName + "/SAAS_BOOST_STACK not found setting to default 'sb-" + this.envName + "'");
+            stackName = "sb-" + this.envName;
```

Comment:
It would be good to start maintaining all the setup defaults we have sprinkled throughout the installer in a single Defaults class or Enum to avoid potential future regressions.

Reply:
Arguably this should match the values in the SettingsService REQUIRED_PARAMS and READ_WRITE_PARAMS collections, but as the notes in the installer say we have some clean up to do on who "owns" some of these parameters (the installer, CloudFormation, the Settings service). Definitely some chicken and egg challenges. At least this path of code in the installer should only execute on an existing install.